apps/src/code-studio/pd/workshop_dashboard/attendance/session_attendance_row.jsx

```diff
@@ -184,6 +187,12 @@ export default class SessionAttendanceRow extends React.Component {
               : null}
           </td>
         )}
+        {this.props.scholarshipWorkshop && (
+          <td>{this.props.attendance.cdo_scholarship ? 'Yes' : ''}</td>
+        )}
+        {this.props.scholarshipWorkshop && (
```

Comment:
Can we also combine these two?

Also, would it help simplify things if we just passed a single enumeration prop that can be "yes/code.org scholarship/other scholarship" rather than relying on a combination of props?

Reply:
Oh, is it possible to have both a code.org scholarship and an other scholarship?  Or are they mutually exclusive?

Reply:
I had that originally, but it felt more complicated, plus I would have to restructure the constants file to get them to the server to be compared.

But yes we can combine!

Reply:
they are mutually exclusive, but Tawny requested this column structure for the CSV, which is what she uses to calculate payments. On the main workshop details page, there is one single column that displays either the scholarship dropdown or just the scholarship status as a string like "Yes, Code.org" or "Yes, other". (you see the dropdown or string based on user permissions)

Reply:
This page is where you download the CSV, so I am keeping the table in sync with what you see on the CSV